### ISSUE TITLE
Could you give test_data.txt for look?

### ISSUE THREAD

**Issue by Darinyazanr** [drcut/FFM-tensorflow#1]

**Comment by Dellen** [comment]
+1

**Comment by Sandy4321** [comment]
yes it would be great to share file with data, everybody puts some data to demonstrate how code works by the way this link 
https://tech.meituan.com/deep-understanding-of-ffm-principles-and-practices.html
in Chinese, do you similar link in English?

**Comment by gtbai** [comment]
+1

**Comment by gtbai** [comment]
In case you guys are still in need, I found `test_data.txt` in [one of the previous commits](https://github.com/drcut/FFM-tensorflow/blob/0ab97c7b7000041abb5bff22406e3631f3dff2b9/test_data.txt).

**Comment by stubird** [comment]
The data format is same as this project:
https://github.com/guestwalk/libffm

Data Format
===========

The data format of LIBFFM is:

<label> <field1>:<feature1>:<value1> <field2>:<feature2>:<value2> ...

**Comment by stubird** [comment]
The data format is same as this project:
https://github.com/guestwalk/libffm

Data Format
===========

The data format of LIBFFM is:

\<label\> \<field1\>:\<feature1\>:\<value1\> \<field2\>:\<feature2\>:\<value2\> ...

**Comment by Sandy4321** [comment]
great you can find data here, at previous commit
https://github.com/drcut/FFM-tensorflow/blob/0ab97c7b7000041abb5bff22406e3631f3dff2b9/test_data.txt

**Comment by Sandy4321** [comment]
thanks to everyone for help

**Comment by Sandy4321** [comment]
if somebody knows more data in format for FFM?
better with original raw data and transformed data
